### ISSUE TITLE
Unexpected rendering of difftime after calling View()

### ISSUE THREAD

**Issue by ColinFay** [rstudio/rstudio#4135]
### System details

    RStudio Edition : Desktop
    RStudio Version : Version 1.2.1147
    OS Version      : macOS Mojave 10.14
    R Version       : `R version 3.4.4 (2018-03-15)`

reproduced here : https://rstudio.cloud/project/165981 

### Steps to reproduce the problem

``` r 
start <- seq.POSIXt(
  as.POSIXct("2017-01-01 00:00:00"), 
  as.POSIXct("2017-01-02 00:00:00"), 
  length.out = 10
)

end <- seq.POSIXt(
  as.POSIXct("2017-01-03 00:00:01"), 
  as.POSIXct("2017-01-04 00:00:03"), 
  length.out = 10
)

df <- data.frame(
  start = start, 
  end = end, 
  diff = end - start
)
df
View(df)

```

### Describe the problem in detail

The difftime object doesn't render well when called with View()

![screenshot 2019-01-10 at 13 55 45](https://user-images.githubusercontent.com/17936236/50969653-741f9480-14df-11e9-8494-866ec070deef.png)

Doesn't seem to be related to my Rstudio version as rstudio.cloud display the same : 

![screenshot 2019-01-10 at 13 58 12](https://user-images.githubusercontent.com/17936236/50969786-db3d4900-14df-11e9-9d47-86f0ba6278f0.png)

### Describe the behavior you expected

I'm not sure where the numbers displayed in the panel come from. 

I would have expected either the same output as in the console or a as.numeric() of the difftime. 

FWIW, here is the output when using R GUI: 

![screenshot 2019-01-10 at 14 00 08](https://user-images.githubusercontent.com/17936236/50969913-29524c80-14e0-11e9-875f-0273f6dbda61.png)

**Comment by ronblum** [comment]
@ColinFay Thank you for reporting this issue. We'll review it as we work on ongoing development of RStudio.

**Comment by kevinushey** [comment]
We might want to consider calling `format()` or similar on each column to get the appropriate representation.

**Comment by mikebessuille** [comment]
Consider this for data viewer improvements in the future.

**Comment by kevinushey** [comment]
Looks like it's already fixed (at least in 1.4-JR)

<img width="425" alt="Screen Shot 2021-02-22 at 4 41 32 PM" src="https://user-images.githubusercontent.com/1976582/108788441-d35c8980-752c-11eb-98da-cc0409fad215.png">
